testing/run-scripts/run_cloud.sh

```diff
@@ -188,4 +188,20 @@ if ! docker ps -a | grep uproxy-sshd >/dev/null; then
   fi
 fi
 
+echo "CLOUD_INSTALL_PROGRESS 95"
+
+# Boot monitoring servers and metrics.
```

Comment:
done! Should I call run_cloud.sh in install_cloud.sh, or just have instructions somewhere that we should call run_monitor.sh manually?